src/common/create-error/plural-module-presence.js

```diff
@@ -0,0 +1,9 @@
+define([
+	"../create-error"
+], function( createError ) {
+
+return function() {
+	return createError( "E_MISSING_PLURAL_MODULE", "Plural module not loaded." );
```

Comment:
As far as I can tell this is used exactly once. I don't think that's worth the extra file.